client/my-sites/theme/main.jsx

```diff
@@ -41,7 +41,8 @@ import {
 	isThemePremium,
 	isThemePurchased,
 	getThemeRequestErrors,
-	getThemeForumUrl
+	getThemeForumUrl,
+	isWpcomTheme as isThemeWpcom,
```

Comment:
Hmm. Can we simply call the prop `isWpcom` (much like we have `isActive` corresponding to the `isThemeActive` selector)? Would save us some aliasing which personally I find rather confusing...